ArduPlane/ArduPlane.cpp

```diff
@@ -52,6 +52,9 @@ SCHED_TASK_CLASS arguments:
  */
 const AP_Scheduler::Task Plane::scheduler_tasks[] = {
                            // Units:   Hz      us
+	SCHED_TASK(update_state, 		    1,     50,   3),
```

Comment:
Any particular reason for these frequencies? If not, could you add a TODO comment to help with tracking